Formula/counterfeiter.rb

```diff
@@ -21,7 +21,7 @@ def install
 
   test do
     output = shell_output("#{bin}/counterfeiter -p os 2>&1", 1)
-    assert_match "Writing `Os` to `osshim`...", output
+    assert_match "go: cannot find GOROOT directory", output
```

Comment:
This looks like a legit error in the test

Reply:
yeah, sounds about right, let me give another try.